enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -122,18 +122,20 @@
           updated-rows (volatile! [])]
       (doseq [row rows]
         (let [;; well, this is a trick, but I haven't figured out how to do single row update
-              result        (:rows-updated (perform-bulk-action! :bulk/update table-id [row]))
-              after-row     (-> (query-db-rows table-id pk-field [row]) vals first)
-              row-before    (get id->db-row (get-row-pk pk-field row))
-              [_ changes _] (diff row-before row)]
+              result     (:rows-updated (perform-bulk-action! :bulk/update table-id [row]))
+              after-row  (-> (query-db-rows table-id pk-field [row]) vals first)
+              row-before (get id->db-row (get-row-pk pk-field row))]
           (vswap! updated-rows conj after-row)
           (when (pos-int? result)
-            (events/publish-event! :event/data-editing-row-update
-                                   {:table_id table-id
-                                    :after    after-row
-                                    :before   row-before
-                                    :update   changes
-                                    :actor_id api/*current-user-id*}))))
+            (events/publish-event! :event/action.success
```

Comment:
Still publishing these manually for two reasons:

1. I haven't used the calling-an-action-within-an-action stuff to publish events yet.
2. I haven't moved all this logic to publish the event down into the row/* actions yet.

I'm actually balking a bit on (2). It'll be nice for "model" events to update the audit log too, but this is turning into a big time sink for now.

I'm planning to wrap the `:bulk/*` events in `:table.edit/*` events, but those will be bulk oriented so subscribing to those probably doesn't work.